setup.cfg

```diff
@@ -37,13 +37,19 @@ install_requires =
 
     # default numpy requirements
     numpy==1.13.3; python_version=='3.5' and platform_machine!='aarch64' and platform_system!='AIX'
-    numpy==1.13.3; python_version=='3.6' and platform_machine!='aarch64' and platform_system!='AIX'
-    numpy==1.14.5; python_version=='3.7' and platform_machine!='aarch64' and platform_system!='AIX'
-    numpy==1.17.3; python_version=='3.8' and platform_machine!='aarch64'
-    numpy==1.19.3; python_version=='3.9'
+    numpy==1.13.3; python_version=='3.6' and platform_machine!='aarch64' and platform_system!='AIX' and platform_python_implementation != 'PyPy'
+    numpy==1.14.5; python_version=='3.7' and platform_machine!='aarch64' and platform_system!='AIX' and platform_python_implementation != 'PyPy'
+    numpy==1.17.3; python_version=='3.8' and platform_machine!='aarch64' and platform_python_implementation != 'PyPy'
+    numpy==1.19.3; python_version=='3.9' and platform_python_implementation != 'PyPy'
+
+    numpy==1.19.0; python_version=='3.6' and platform_python_implementation=='PyPy'
+    # Change this to 1.20 after it is released since 1.20 will be the first
+    # wheel version on PyPI
+    numpy==1.19.0; python_version=='3.7' and platform_python_implementation=='PyPy'
 
     # For Python versions which aren't yet officially supported,
     # we specify an unpinned Numpy which allows source distributions
     # to be used and allows wheels to be used as soon as they
     # become available.
     numpy; python_version>='3.10'
+    numpy; python_version>='3.8' and platform_python_implementation=='PyPy'
```

Comment:
I probably would have suggested setting this to Python 3.7 and putting the 3.7 spec in a separate PR, but that ship has sailed already :sweat_smile:

Reply:
Well, pypy-37 is out already, so it's not really a future version. So I think it's fine. I added a step to the NumPy release howto to update this package whenever there's a new Python version or new wheels for some platform - so we'll get a reminder as soon as 1.20.0 is out.